src/components/lot/QuickBidModal.tsx

```diff
@@ -0,0 +1,202 @@
+import { useState } from "react";
+import Image from "next/image";
+import { usePlaceBidMutation } from "@hooks";
+import { strings, images } from "@constants";
+import styled from "styled-components";
+import { Button } from "@components";
+import { ILotData } from "@interfaces";
+
+const Modal = styled.div(
+  ({ theme }) => `
+  background: ${theme.colors.modalOverlayBackground};
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+`
+);
+
+const ModalContent = styled.section(
+  ({ theme }) => `
+  position: fixed;
+  background: ${theme.colors.background};
+  width: 60%;
+  max-width: ${theme.breakpoints.md}px;
+  max-height: 90vh;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border-radius: ${theme.borderRadius.medium};
+  padding: 45px 61px;
+  overflow-y: auto;
+  text-align: center;
+  ${theme.down(theme.breakpoints.md)} {
+    padding: 45px 15px;
+    width: 90%;
+  }
+`
+);
+
+const ModalTitle = styled.h3(
+  ({ theme }) => `
+  margin-top: 0;
+  margin-bottom: 40px;
+  text-align: center;
+  font-size: 24px;
+`
+);
+
+const ModalSubTitle = styled.h4(
+  ({ theme }) => `
+  margin-top: 0;
+  margin-bottom: 14px;
+  text-align: center;
+  font-size: 32px;
+`
+);
+
+const BidAmountText = styled.p(
+  ({ theme }) => `
+  font-family: "PostGrotesk";
+  font-style: normal;
+  font-size: 20px;
+  line-height: 22px;
+  text-align: center;
+`
+);
+
+const BidAmount = styled.span(
+  ({ theme }) => `
+  font-weight: 600;
+  color: #29ABE2;
+`
+);
+
+const ButtonsContainer = styled.div(
+  ({ theme }) => `
+  display: flex;
+  justify-content: space-evenly;
+  margin-top: 44px;
+  margin-bottom: 30px;
+  grid-gap: 24px;
+`
+);
```

Comment:
```suggestion
const ButtonsContainer = styled.div`
  display: flex;
  justify-content: space-evenly;
  margin-top: 44px;
  margin-bottom: 30px;
  grid-gap: 24px;
`;
```
Also, why the `grid-gap` in a flex container?